osu.Game/Screens/OnlinePlay/Components/RoomManager.cs

```diff
@@ -117,7 +113,7 @@ public void AddOrUpdateRoom(Room room)
             try
             {
                 foreach (var pi in room.Playlist)
-                    pi.MapObjects(rulesets);
+                    pi.MapObjects();
```

Comment:
I guess removing this completely is a follow-up step?

Reply:
Correct, it'll be part of the PR removing the beatmap bindable.